### ISSUE TITLE
Add the ability to add and hotels and view them in the frontend

### ISSUE THREAD

**Issue by bartspreij** [TimW90/BookingApp#11]
### What is the feature or task?

We would like the ability to add hotels through a popup menu in the frontend that then gets persisted in the backend. 

### Importance and Impact

For our app to provide a wide array of hotels we need the ability to add new hotels.

### Expected Outcome

A pop up menu with a form to add a hotel to the app. In the form you will be able to provide details about the new hotel and upload an image. 

### Implementation Details

Basically a form that sends a Hotel to the backend and will show in the frontend afterwards.

### Additional Context

gogo
